lib/galaxy/managers/workflows.py

```diff
@@ -535,7 +541,12 @@ def _workflow_to_dict_instance(self, trans, stored):
                     label = "Input Dataset Collection"
                 else:
                     raise ValueError("Invalid step_type %s" % step_type)
-                inputs[step.id] = {'label': label, 'value': ""}
+                if legacy:
+                    index = step.id
+                else:
+                    index = step.order_index
+                uuid = str(step.uuid) if step.uuid else None
```

Comment:
Can you change the variable name from `uuid` to `step_uuid`?
I wouldn't use the name of standard library Python module for a variable, even if it is not used in this file.